roles/nameserver/tasks/firewall.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Enable firewalld
```

Comment:
firewalld is the default firewall in CentOS 7.  The iptables service isn't installed by default anymore.